results_spreadsheet.py

```diff
@@ -64,14 +64,10 @@ def clean_and_aggregate(df, scenario, region_to_state):
 
     # aggregate value
     # Identify numeric columns to aggregate
-    health_cols = ["tons", "md", "damage_$", "mortality"]
+    value_cols = ["value_emit", "value", "Value", "tons", "md", "damage_$", "mortality"]
```

Comment:
Is it necessary to have `"value"` and `"Value"` if earlier in the script you force all column names to lowercase with `str.lower()`?